examples/create_rosa_cluster/create_rosa_sts_cluster/oidc_configuration/cluster_with_managed_oidc_config/README.md

```diff
@@ -1,43 +1,66 @@
-# ROSA STS cluster creation example with a managed OIDC configuration ID
+# Creating a ROSA STS cluster with a managed OIDC configuration ID
 
-This example shows how to create managed OIDC config an operator IAM roles and OIDC provider before creating a cluster.
+This Terraform example creates a ROSA STS cluster that uses a managed OIDC configuration. For more information on OIDC configurations, see the [OpenID Connect Overview](https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/introduction_to_rosa/rosa-oidc-overview) in the Red Hat Customer Portal.
 
-To run it:
+## Prerequisites
 
-* Provide OCM Authentication Token
+* You have an AWS account.
+* You have installed the latest version ROSA CLI (`rosa`)
+* You must have an offline OCM token. This token can be generated in the [Red Hat Hybrid Cloud Console](https://console.redhat.com/openshift/token).
+* You have installed Terraform. See the [Hashicorp Terraform page](https://developer.hashicorp.com/terraform/downloads) for the latest version.
+* You have created your [account-wide IAM Roles](https://github.com/terraform-redhat/terraform-provider-ocm/blob/a42779d6b6712f4dde358344f44b782e4dfcd120/examples/create_rosa_cluster/create_rosa_sts_cluster/classic_sts/account_roles/README.md). For more information, see [Account-wide IAM role and policy reference](https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/introduction_to_rosa/rosa-sts-about-iam-resources#rosa-sts-account-wide-roles-and-policies_rosa-sts-about-iam-resources) in the Red Hat Customer Portal.
 
-    OCM authentication token that you can get [here](https://console.redhat.com/openshift/token).
 
-    ```
-    export TF_VAR_token=...
-    ```
+## ROSA cluster creation
 
-* Provide OCM environment by setting a value to url
+1. To run the `terraform apply` you need to set up various variables. This guide uses environmental variables. For a more expansive explanation of Terraform variables, see [Managing Variables](https://developer.hashicorp.com/terraform/enterprise/workspaces/variables/managing-variables) in the Terraform documentation.
```

Comment:
See above notes on line 14 of the previous module about "various variables" and "expansive explanation".